src/libraries/System.Security.Cryptography/src/System/Security/Cryptography/CryptoStream.cs

```diff
@@ -719,9 +719,9 @@ public override Task CopyToAsync(Stream destination, int bufferSize, Cancellatio
         private async Task CopyToAsyncInternal(Stream destination, int bufferSize, CancellationToken cancellationToken)
         {
             // Use ArrayPool<byte>.Shared instead of CryptoPool because the array is passed out.
-            byte[]? rentedBuffer = ArrayPool<byte>.Shared.Rent(bufferSize);
+            byte[] rentedBuffer = ArrayPool<byte>.Shared.Rent(bufferSize);
             // Pin the array for security.
-            GCHandle pinHandle = GCHandle.Alloc(rentedBuffer, GCHandleType.Pinned);
+            using PinnedGCHandle<byte[]> pinHandle = new PinnedGCHandle<byte[]>(rentedBuffer);
```

Comment:
```suggestion
            PinnedGCHandle<byte[]> pinHandle = new PinnedGCHandle<byte[]>(rentedBuffer);
```

Reply:
I've reverted all changes in `CryptoStream`.